src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensHologram.java

```diff
@@ -220,11 +220,8 @@ private void updateHolograms() {
                 for (final Player player : Bukkit.getOnlinePlayers()) {
                     boolean visible = true;
 
-                    for (final ConditionID condition : npcHologram.config.conditions) {
-                        if (!BetonQuest.condition(PlayerConverter.getID(player), condition)) {
-                            visible = false;
-                            break;
-                        }
+                    if (!BetonQuest.conditions(PlayerConverter.getID(player), npcHologram.config.conditions)) {
+                        visible = false;
```

Comment:
With `boolean visible = true` above (can't be included into suggestions) this if isn't required but instead `visible` can be set directly:

```suggestion
                    visible = BetonQuest.conditions(PlayerConverter.getID(player), npcHologram.config.conditions);
```

It might be possible to completely omit `visible` if the check directly below is the only one that uses visible.